.travis.yml

```diff
@@ -7,8 +7,12 @@ php:
   - nightly
   - hhvm
 
-install:
+before_script:
   - composer self-update
-  - composer install --prefer-source
+  - composer install --dev --prefer-source
+  - mkdir -p build/logs
```

Comment:
Being super picky here: I read the documentation on the php-coveralls website: https://github.com/satooshi/php-coveralls#travis-ci

Can you update this file to match what they have?

```
before_script:
 - wget http://getcomposer.org/composer.phar
 - php composer.phar install --dev --no-interaction

script:
 - mkdir -p build/logs
 - phpunit --coverage-clover build/logs/clover.xml

after_script:
 - php vendor/bin/coveralls -v
```